src/Core/src/Platform/Windows/ImageSourcePartExtensions.cs

```diff
@@ -42,8 +42,6 @@ internal static class ImageSourcePartExtensions
 				if (applied)
 				{
 					setImage(uiImage);
-					if (destinationContext is WImage imageView)
-						imageView.UpdateIsAnimationPlaying(image);
```

Comment:
This did nothing as the image is only animatable after it is decoded - which is after it i on-screen.